buildhub/main/management/commands/reindex-bigquery.py

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import requests
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+from buildhub.main.models import Build
+
+
+class Command(BaseCommand):
+    help = "Reindex tables from Postgres into BigQuery via batched inserts."
+
+    def handle(self, *args, **options):
+        """See https://googleapis.dev/python/bigquery/latest/index.html."""
+
+        # TODO: Best place to put this variable? Implicit in GOOGLE_APPLICATION_CREDENTIALS
+        project_id = "buildhub2-bigquery-dev"
```

Comment:
`settings.py` seems like it would the most appropriate place. It looks like this can be set explicitly through the `Client` API: https://googleapis.dev/python/bigquery/latest/generated/google.cloud.bigquery.client.Client.html#google.cloud.bigquery.client.Client. I think it's fine to allow credentials to be configured through `GOOGLE_APPLICATION_CREDENTIALS`, but I'd prefer `project` to be explicit.

Most likely we'd want to do it for at least stage and prod (but maybe dev too?), so perhaps part of https://github.com/mozilla-releng/buildhub2/blob/ee348d79d646b689bd91f1f41dee65e857d16a4b/buildhub/settings.py#L195, and have that read from the environment?

Reply:
Added the relevant settings, `BQ_PROJECT_ID`, `BQ_DATASET_ID`, and` BQ_TABLE_ID`.